src/test/java/com/epam/ta/steps/PostSteps.java

```diff
@@ -0,0 +1,60 @@
+package com.epam.ta.steps;
+
+import com.epam.ta.model.post.Post;
+import org.assertj.core.api.AutoCloseableSoftAssertions;
+
+public class PostSteps {
+
+    public static void verifyPost(Post actualPost, Integer id, Integer userId, String title, String body) {
+        try (AutoCloseableSoftAssertions softly = new AutoCloseableSoftAssertions()) {
```

Comment:
tbh, try to do it without try expression (using .assertAll())
https://joel-costigliola.github.io/assertj/core/api/org/assertj/core/api/SoftAssertions.html

Reply:
softly.assertAll();

Reply:
try with resources